integration/spark2/src/main/scala/org/apache/carbondata/spark/util/CarbonSparkUtil.scala

```diff
@@ -117,4 +116,18 @@ object CarbonSparkUtil {
     case _ =>
       delimiter
   }
+  def getKeyOnPrefix(path: String): (String, String, String) = {
```

Comment:
getKeyOnPrefix is the same as S3Example, why add the same method ,but not be called